doc/conf.py

```diff
@@ -15,10 +15,15 @@
 
 import datetime
 import os
+import pathlib
 import subprocess
 import sys
 from contextlib import suppress
 
+root = pathlib.Path(__file__).absolute().parent.parent
```

Comment:
Do you want to add a comment here referencing this / the original issue, so it's not removed in future?

Reply:
done?